packages/foundation-models/src/azure-openai/azure-openai-line-decoder.ts

```diff
@@ -0,0 +1,105 @@
+type Bytes = string | ArrayBuffer | Uint8Array | Buffer | null | undefined;
```

Comment:
[req] I would put the streaming specific files into a separate folder.

Reply:
Moved.